### PR TITLE
Add fluent-icon-theme and fluent-theme

### PR DESCRIPTION
**Pull request by lleyton**
As the title says. Also added some devcontainer updates.

### CODE REVIEW

File: anda/themes/fluent-icon-theme/anda.hcl [hunk 1]
```diff
@@ -0,0 +1,5 @@
+project "pkg" {
+    rpm {
```
Comment (madonuko) [inline anda/themes/fluent-icon-theme/anda.hcl]:
```suggestion
project "pkg" {
    arches = ["x86_64"]
    rpm {
```

File: anda/themes/fluent-theme/anda.hcl [hunk 2]
```diff
@@ -0,0 +1,5 @@
+project "pkg" {
+    rpm {
```
Comment (madonuko) [inline anda/themes/fluent-theme/anda.hcl]:
```suggestion
project "pkg" {
    arches = ["x86_64"]
    rpm {
```

File: anda/themes/fluent-theme/fluent-theme.spec [hunk 3]
```diff
@@ -0,0 +1,38 @@
+%global tag 2022-12-15
+
+Name:           fluent-theme
+Version:        20221215
+Release:        1%{?dist}
+Summary:        Fluent is a Fluent design theme for GNOME/GTK based desktop environments.
+
+License:        GPLv3
```
Comment (madonuko) [inline anda/themes/fluent-theme/fluent-theme.spec]:
```suggestion
License:        GPL-3.0
```

File: anda/themes/fluent-theme/update.rhai [hunk 4]
```diff
@@ -0,0 +1,3 @@
+let release = gh("vinceliuice/Fluent-gtk-theme")
+rpm.global("commit", sha);
```
Comment (madonuko) [inline anda/themes/fluent-theme/update.rhai]: `sha` is undefined?

Comment (madonuko) [reply anda/themes/fluent-theme/update.rhai]: DID YOU FORGET SEMICOLON COME ON LEA WE HAVE THE BEST DEVDOCS IN THE WORLD

**Review by madonuko — CHANGES_REQUESTED** [review]

File: anda/themes/fluent-theme/update.rhai (hunk 4 above)
Comment (lleyton) [reply anda/themes/fluent-theme/update.rhai]: fuck, I am so out of it tonight

**Review by madonuko — APPROVED** [review]

**Pull request merged**